twitter.html

```diff
@@ -0,0 +1,227 @@
+                    <img class="avatar1" src="/web-images/avatar1.png" alt="">
+                    <div class="statement-body">
+                        <div class="profile-statement two-column-grid">
+                            <p>Cliff<span class="user">@Users Name</span></p>
+                        </div>
+                        <p class="paragraph"> Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+                            Suscipit delectus molestiae, quo sunt incidunt ab repellat vel 
+                            repellendus maiores magnam neque illum voluptatem, aliquid dolorum voluptate 
+                            facilis dolor perferendis dolores.</p>
+                            <div class="statement-icons grid-auto__column">
+                                <img src="/web-images/comment.png" alt="">
+                                <img src="/web-images/gif.png" alt="">
+                                <img src="/web-images/heart.png" alt="">
+                                <img src="/web-images/upload.png" alt="">
+                            </div>
+                    </div>
+                    <div >
+                        <img class="right-icon" src="/web-images/3dots.png" alt="">
+                    </div>
+                    </div>
+
+        </div>
+        <div class="boxes">
+                    <div class=" three-column-grid__expand-two">
+        
+                        <img class="avatar1" src="/web-images/avatar1.png" alt="">
+                    
+                        <div class="statement-body">
+                            <div class="profile-statement two-column-grid">
+                                <p>ThatBlonde Guy<span class="user">@Users Name</span></p>
+                            </div>
+                            <p class="paragraph" > Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+                                Suscipit delectus molestiae, quo sunt incidunt ab repellat vel 
+                                repellendus maiores magnam neque illum voluptatem, aliquid dolorum voluptate 
+                                facilis dolor perferendis dolores.</p>
+                                <div class="statement-icons grid-auto__column">
+                                    <img src="/web-images/comment.png" alt="">
+                                    <img src="/web-images/gif.png" alt="">
+                                    <img src="/web-images/heart.png" alt="">
+                                    <img src="/web-images/upload.png" alt="">
+                                </div>
+                        </div>
+                        <div>
+                            <img class="right-icon" src="/web-images/3dots.png" alt="">
+                        </div>
+                        </div>
+
+            </div>  
+            
+            
+        </div>
+        
+
+            
+
+            
+        
+
```

Comment:
Have you downloaded the prettier extension on VSCode? I think this big gap not needed.